app/code/Magento/Newsletter/Controller/Subscriber/NewAction.php

```diff
@@ -10,19 +10,31 @@
 use Magento\Customer\Model\Session;
 use Magento\Customer\Model\Url as CustomerUrl;
 use Magento\Framework\App\Action\Context;
+use Magento\Framework\App\Config\ScopeConfigInterface;
+use Magento\Framework\Exception\LocalizedException;
+use Magento\Framework\Phrase;
+use Magento\Framework\Validator\EmailAddress;
```

Comment:
I think it's better to use alias `EmailAddressValidator` or `EmailValidator` there to be clear